src/ui/server/graphql/apollo.ts

```diff
@@ -38,6 +38,11 @@ const doQuery = async (query: DocumentNode, variables: OperationVariables) => {
     return await client.query({
       query,
       variables,
+      context: {
```

Comment:
Just to confirm, the whole frontend server runs on a graphql API that wasn't send an API key before so everything is auto covered by attaching it to every query/mutation?

Reply:
Yes exactly 😄 All calls to our API from the UI are made via GraphQL, apollo client. So every call will go through this file or more specifically here: https://github.com/UK-Export-Finance/exip/blob/c52d1284cbba3ca3b3a7b879ede69ff5fd02473c/src/ui/server/graphql/apollo.ts#L69